tensorboard/webapp/metrics/views/main_view/main_view_component.scss

```diff
@@ -160,3 +165,25 @@ limitations under the License.
     margin-right: 4px;
   }
 }
+
+.slide-out-menu {
+  background-color: mat.get-color-from-palette($tb-background, background);
+  height: 100%;
+  position: absolute;
+  right: 50px;
+  top: 50px;
```

Comment:
Where does this `top: 50px` come from? It looks like there is a gap
![image](https://user-images.githubusercontent.com/78179109/213262185-57dc1399-da32-4985-a975-5d4112511e5d.png)

Reply:
So I have determined that it should be 49px. The regex filter bar is set to 48px with a bottom border of 1 px. The gap you have there is really big so not sure what how that happened but it looks good for me.